spec/cucumber/cli/profile_loader_spec.rb

```diff
@@ -42,6 +42,16 @@ def loader
         given_cucumber_yml_defined_as yml
         expect(loader.args_from('default')).to eq ['--format','pretty','features/sync_imap_mailbox.feature:16:22']
       end
+
+      it 'correctly parses a profile that uses tag expressions' do
```

Comment:
I don't know why this PR isn't noticing the new commit that I made. The new test has been added to this branch but it's not showing up...